src/bin/bitmaskd.rs

```diff
@@ -67,6 +85,7 @@ async fn main() -> Result<()> {
     pretty_env_logger::init();
 
     let app = Router::new()
+        .route("/issue-contract", post(issue_contract))
```

Comment:
Please don't make this a new route, just replace the /issue route. It'd reduce maintenance complexity.

Reply:
No. We haven't even hit 1.0.